packages/eslint-config-base/rules/imports.js

```diff
@@ -93,6 +93,14 @@ module.exports = {
         ],
       },
     ],
+    /**
+     * Use named exports over default exports
+     * @link https://github.com/reside-eng/guidelines/blob/main/rfcs/shared-patterns/typescript/001-consistent-import-export.md
+     */
+    ...{
```

Comment:
I still think there's a way around this. For example, maybe we could start a `guidelines` folder that has `001-consistent-import-export.js`. The `index.js` for the `guidelines` could take all the accepted guidelines and spread them over an export. This would make the import into `rules/imports.js` one line of `const guidelines = require('./guidelines')` then, right here on 100, it would just be `...guidelines`.

Reply:
@hershmire is this reorganization of rules based on guidelines something you want us to start doing?

Reply:
I discussed with @offgriddev and we're on the same page. This file organization is what we want to go with. However, maybe to make things a little easier to read/maintain, we don't do the object spreading like you're doing. We can just group these two lines of rules next to one another with the JS docs comment.

Reply:
Sounds good; I removed the object spread syntax.